docs/index.txt

```diff
@@ -14,12 +14,8 @@ designed to facilitate packaging Python projects, where packaging includes:
 Documentation content:
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-   setuptools
-   pkg_resources
-   python3
-   development
-   roadmap
-   Deprecated: Easy Install <easy_install>
-   history
```

Comment:
Was the removal of the changelog deliberate?

Reply:
No. The changelog should be retained, and it perhaps should even have a forwarding link so historical links can still reach it.

Reply:
FWIW I've sent a few recovery PRs like https://github.com/pypa/setuptools/pull/2425